moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -217,6 +223,7 @@ def robot_description(
             robot_description_file_path = self.__urdf_package / self.__urdf_file_path
         else:
             robot_description_file_path = self._package_path / file_path
+        mappings = self.__urdf_xacro_args
```

Comment:
This will always override the mappings that the user passes, you should move this to line#234 `robot_description_file_path, mappings=mappings or self.__urdf_xacro_args`